src/FSharpVSPowerTools.Core/Utils.fs

```diff
@@ -1,4 +1,42 @@
-﻿namespace FSharpVSPowerTools
+﻿namespace FSharpPowerTools.Core
+
+type FileName = string
+
+[<Measure>] type FCS
+
+type Point<[<Measure>]'t> = { Line : int; Column : int }
+type Range<[<Measure>]'t> = { From : Point<'t>; To: Point<'t> }
+
+[<CompilationRepresentation(CompilationRepresentationFlags.ModuleSuffix)>]
+module Point =
+    let make line column : Point<'t> = { Line = line; Column = column }
+
+[<CompilationRepresentation(CompilationRepresentationFlags.ModuleSuffix)>]
+module Range =
+
+    let make startLine startColumn endLine endColumn : Range<'t> =
+        {
+          From = Point.make startLine startColumn
+          To = Point.make endLine endColumn 
+        }
+    
+type CurrentLine<[<Measure>]'t> = { Line: string; File: FileName; Range: Range<'t> }
+    with
+        member x.EndLine = x.Range.To.Line 
+
+[<NoComparison>]
+type PointInDocument<[<Measure>]'t> = {
+  Point    : Point<'t>
+  Line     : string
+  Document : string
+  File     : FileName
+}
```

Comment:
This type is very similar to `CurrentLine`. What about embed the latter to it? Like:
```fsharp
type PointInDocument =
    { Line: CurrentLine
      Document: string }
```